html.go

```diff
@@ -748,7 +748,7 @@ func (r *HTMLRenderer) writeDocumentHeader(w *bytes.Buffer, sr *SPRenderer) {
 	w.WriteString(">\n")
 	if r.CSS != "" {
 		w.WriteString("  <link rel=\"stylesheet\" type=\"text/css\" href=\"")
-		attrEscape([]byte(r.CSS))
+		w.Write(attrEscape([]byte(r.CSS)))
```

Comment:
It looks like most other code uses `esc` instead of calling `attrEscape` directly.

Also, both `esc` and `escCode` take 2 parameters, but the second parameter is never used. That's strange. /cc @rtfb